src/components/ChapterRequest.tsx

```diff
@@ -0,0 +1,152 @@
+"use client";
+import { useState } from "react";
+import { faEnvelope, faPhone } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { handleChapterRequest } from "src/app/api/handle-chapter-request/route.client";
+
+type ButtonProps = {
+  chapterRequestId: string;
+  universityName: string;
+  universityAddress: string;
+  name: string;
+  email: string;
+  phoneNumber: string;
+  leadershipExperience: string;
+  motivation: string;
+  availabilities: string;
+  questions: string;
+};
+
+const ChapterRequest = ({
+  chapterRequestId,
+  universityName,
+  universityAddress,
+  name,
+  email,
+  phoneNumber,
+  leadershipExperience,
+  motivation,
+  availabilities,
+  questions,
+}: ButtonProps) => {
+  const [showMore, setShowMore] = useState(false);
+  return (
+    <div className="grid h-min w-full rounded-lg bg-white p-6 text-xs">
+      <div className="mb-1 text-xl underline">
+        {universityName} <br />
+      </div>
+      <div className="flex ">
+        <div className="flex w-6/12 flex-col ">
+          <div className="mb-2.5">
+            Location: <span className="font-bold">{universityAddress} </span>
+          </div>
+          <div className="mb-2.5">
+            Requester:&nbsp;<span className="font-bold">{name} </span>
+          </div>
+        </div>
+        <div className="flex w-6/12 flex-col">
+          <div className="mb-2.5">
+            <FontAwesomeIcon icon={faEnvelope} className="mr-1" />
+            {email} <br />
+          </div>
+          <div className="mb-2.5">
+            <FontAwesomeIcon icon={faPhone} className="mr-1" />
+            {phoneNumber} <br />
+          </div>
+        </div>
+      </div>
+
+      {showMore ? (
+        <div>
+          <div
+            className="text-dark-green underline"
```

Comment:
We can consider adding `cursor-pointer` here for better feedback on the application